scriptmodules/libretrocores/lr-pcsx-rearmed.sh

```diff
@@ -48,7 +48,7 @@ function configure_lr-pcsx-rearmed() {
     ensureSystemretroconfig "psx"
 
     # system-specific, PSX
-    iniConfig " = " "" "$configdir/psx/retroarch.cfg"
+    iniConfig " = " '"' "$configdir/psx/retroarch.cfg"
     iniSet "rewind_enable" "false"
```

Comment:
rewind_enable is already set to false system-wide by default. so the line can be removed.